src/app/+statistics-page/collection-statistics-page/collection-statistics-page.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component } from '@angular/core';
+import { StatisticsPageComponent } from '../statistics-page/statistics-page.component';
+import { UsageReportService } from '../../core/submission/usage-report-data.service';
+import { ActivatedRoute , Router} from '@angular/router';
+import { Collection } from '../../core/shared/collection.model';
+import { DSONameService } from '../../core/breadcrumbs/dso-name.service';
+
+/**
+ * Component representing the statistics page for a collection.
+ */
+@Component({
+  selector: 'ds-collection-statistics-page',
+  templateUrl: '../statistics-page/statistics-page.component.html',
+  styleUrls: ['./collection-statistics-page.component.scss']
+})
+export class CollectionStatisticsPageComponent extends StatisticsPageComponent<Collection> {
```

Comment:
Can we move all these new statistics components/pages under `src/app/statistics-page`?  There's no need for the "+" in the path. See this long standing issue #327

Reply:
Ok, renamed the directory